utils/hct/hctdb_instrhelp.py

```diff
@@ -1502,3 +1604,12 @@ def RunCodeTagUpdate(file_path):
             ]
         for relative_file_path in files:
             RunCodeTagUpdate(pj(hlsl_src_dir, relative_file_path))
+
+    if args.query_dxil or args.query_hlsl:
```

Comment:
Not sure why these have to be in the same conditional block. DXIL will get printed before HLSL regardless and it requires an extra check when calling parse*